src/platform/common/stm32/bus_spi_pinconfig.c

```diff
@@ -184,6 +184,7 @@ const spiHardware_t spiHardware[] = {
         .misoPins = {
             { DEFIO_TAG_E(PA6), GPIO_AF5_SPI1 },
             { DEFIO_TAG_E(PB4), GPIO_AF5_SPI1 },
+            { DEFIO_TAG_E(PG9), GPIO_AF5_SPI1 },
```

Comment:
Would suspect changes in https://github.com/betaflight/betaflight/blob/master/src/platform/STM32/timer_stm32h7xx.c to add timer(s).

Reply:
There is no timer option on this pin, is that what you were suspecting?
```
	"PG9:DCMI_VSYNC"    	:	13,
	"PG9:EVENT-OUT"     	:	15,
	"PG9:FMC_NCE"       	:	12,
	"PG9:FMC_NE2"       	:	12,
	"PG9:I2S1_SDI"      	:	5,
	"PG9:QUADSPI_BK2_IO2"	:	9,
	"PG9:SAI2_FS_B"     	:	10,
	"PG9:SPDIFRX_IN3"   	:	8,
	"PG9:SPI1_MISO"     	:	5,
	"PG9:USART6_RX"     	:	7,
```
![image](https://github.com/user-attachments/assets/08da0268-5f56-4a93-a12f-952828419566)